rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/80_synonyms_from_index.yml

```diff
@@ -187,3 +187,44 @@ setup:
                 search_analyzer: my_analyzer
   - match: { acknowledged: true }
   - match: { shards_acknowledged: false }
+
+---
+"Load empty synonyms set from index for an analyzer":
```

Comment:
No, but it wasn't explicitly tested and I wanted to add coverage since I removed the [fake rule](https://github.com/elastic/elasticsearch/blob/4173feefc2f23284cc2f151d7ff9807f2985f313/modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/SynonymTokenFilterFactory.java#L174) that was added previously.

Reply:
Makes sense, thanks!